### ISSUE TITLE
Error with media files with special chars

### ISSUE THREAD

**Issue by DevPGSV** [fabianonline/telegram_backup#12]
For some reason, some of the files in my conversations have the "double quote" character at the end. When that file is downloaded and saved to a file, it throws an exception ( https://github.com/fabianonline/telegram_backup/blob/master/src/main/java/de/fabianonline/telegram_backup/DownloadManager.java#L288 )

I had some problems trying to compile from source code, so I wasn't able to fix it to send a pull request.

This is what happens when it downloads a file called:   **68382.mp3"**
```
IOException happened while downloading C:\Users\Usuario\.telegram_backup\+<NUMBER>\files\68382.mp3"
java.io.FileNotFoundException: C:\Users\Usuario\.telegram_backup\+<NUMBER>\files\68382.mp3".downloading (El nombre de archivo, el nombre de directorio o la sintaxis de la etiqueta del volumen no son correctos)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(Unknown Source)
        at java.io.FileOutputStream.<init>(Unknown Source)
        at java.io.FileOutputStream.<init>(Unknown Source)
        at de.fabianonline.telegram_backup.DownloadManager.downloadFileFromDc(DownloadManager.java:288)
        at de.fabianonline.telegram_backup.DownloadManager.downloadFile(DownloadManager.java:267)
        at de.fabianonline.telegram_backup.mediafilemanager.DocumentFileManager.download(DocumentFileManager.java:100)
        at de.fabianonline.telegram_backup.DownloadManager._downloadMedia(DownloadManager.java:247)
        at de.fabianonline.telegram_backup.DownloadManager.downloadMedia(DownloadManager.java:202)
        at de.fabianonline.telegram_backup.CommandLineController.<init>(CommandLineController.java:170)
        at de.fabianonline.telegram_backup.CommandLineRunner.main(CommandLineRunner.java:77)
```

Thanks for the tool! Great work!

**Comment by fabianonline** [comment]
That's really strange. Is the double quote the only special char appearing in your file names? I'll fix this in 1.0.6.

**Comment by DevPGSV** [comment]
Yes, I haven't seen files with special chars different that double quote. I have 8 files like that.
Thanks!
